db/migrate/20211216091337_add_password__to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPasswordToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :password_, :string
```

Comment:
Dùng gem bycrypt thì đặt lại tên cho đúng syntax của gem nha em
```suggestion
    add_column :users, :password_digest, :string
```